certified-connectors/RecordedFutureNew/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1285 @@
+{
+  "swagger":"2.0",
+  "info":{
+    "title":"Recorded Future V2",
```

Comment:
@johannesh-recordedfuture Please update your connector folder as per the new connector title